### PR TITLE
Private bot setting

### PR DESCRIPTION
**Pull request by niansa**
Adds private_bot setting which makes the bot ignore any commands that are not from it's owner if set to True. Returns an error message otherwise.

### CODE REVIEW

**Comment by niansa** [comment]
It's ready for review now ;-)

File: xenon/bot.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ def __init__(self, *args, **kwargs):
         if message.author.bot:
             return
 
+        if message.content.startswith(self.config.prefix) and self.config.private_bot == True and self.config.owner_id != message.author.id:
+            await message.channel.send("Only my owner is allowed to use me!")
```
Comment (ThatLifeofGames) [inline xenon/bot.py]: I hate to sound picky but this doesn't really follow the "standard" of all of xenon's other outputs. Take a look at some of the other outputs, you'll see what I mean :)
Usually everything is in embeds and it's never really in "first person", but this is up to Merlin.

Comment (niansa) [reply xenon/bot.py]: Alright, let me find another error message... 🤔

Comment (niansa) [reply xenon/bot.py]: Changed, what do you think?

**Comment by merlinfuchs** [comment]
I think it would make sense to create a selfhost branch, because the hosted version of Xenon doesn't benefit in any way of these changes.

**Comment by niansa** [comment]
Yeah. But I have no idea how to even do that :thinking:

**Comment by merlinfuchs** [comment]
I created a selfhost branch, you should be able to change your pull request to it.

**Comment by niansa** [comment]
Done

**Pull request merged**